formal.tex

```diff
@@ -590,15 +594,18 @@ \subsection{Structural rules}
   \inferrule*{\jdeqtp\Gamma{a}{b}{A} \\ \jdeqtp\Gamma{A}{B}{\UU_i}}{\jdeqtp\Gamma{a}{b}{B}}
 \end{mathparpagebreakable}
 %
-Additionally, for all the type formers below, we assume rules stating that each constructor preserves definitional equality in each of its arguments; for instance, along with the $\Pi$-\rintro\ rule, we assume the rule
+Finally, we assume that judgmental equality is a congruence respected by typing,
+i.e., that each constructor locally preserves judgmental equality in each of its
```

Comment:
```suggestion
i.e., that each constructor preserves judgmental equality in each of its
```
I'm not sure what the word "locally" adds here.  Is there a way for a single constructor to "globally" preserve judgmental equality?